tests/regression/regression_test_helpers.py

```diff
@@ -0,0 +1,139 @@
+    "pill",
+    "screw",
+    "tile",
+    "toothbrush",
+    "transistor",
+    "wood",
+    "zipper",
+]
+
+
+TIME_LOG = {
+    "train_time": "Train + val time (sec.)",
+    "infer_time": "Infer time (sec.)",
+    "export_time": "Export time (sec.)",
+    "export_eval_time": "Export eval time (sec.)",
+    "deploy_time": "Deploy time (sec.)",
+    "deploy_eval_time": "Deploy eval time (sec.)",
+    "nncf_time": "NNCF time (sec.)",
+    "nncf_eval_time": "NNCF eval time (sec.)",
+    "pot_time": "POT time (sec.)",
+    "pot_eval_time": "POT eval time (sec.)",
+}
+
+
+def get_result_dict(task_type: str) -> Dict[str, Any]:
+    result_dict = {task_type: {}}
+    if "anomaly" not in task_type:
+        for label_type in LABEL_TYPES:
+            result_dict[task_type][label_type] = {}
+            for train_type in TRAIN_TYPES:
+                result_dict[task_type][label_type][train_type] = {}
+                for test_type in TEST_TYPES:
+                    result_dict[task_type][label_type][train_type][test_type] = []
+    else:
+        for test_type in TEST_TYPES:
+            result_dict[task_type][test_type] = {}
+            for category in ANOMALY_DATASET_CATEGORIES:
+                result_dict[task_type][test_type][category] = []
+
+    return result_dict
+
+
+def load_regression_config(otx_dir: str) -> Dict[str, Any]:
+    """Load regression config from path.
+
+    Args:
+        otx_dir (str): The path of otx root directory
+
+    Returns:
+        Dict[str, Any]: The dictionary that includes data roots
+    """
+    root_path = Path(otx_dir)
+    with open(root_path / ("tests/regression/regression_config.json"), "r") as f:
+        reg_config = json.load(f)
+    return reg_config
+
+
+def load_regression_configuration(
+    otx_dir: str, task_type: str, train_type: str = "", label_type: str = ""
+) -> Dict[str, Union[str, int, float]]:
+    """Load dataset path according to task, train, label types.
+
+    Args:
+        otx_dir (str): The path of otx root directoy
+        task_type (str): ["classification", "detection", "instance segmentation", "semantic segmentation",
+                            "action_classification", "action_detection", "anomaly"]
+        train_type (str): ["supervised", "semi_supervised", "self_supervised", "class_incr"]
+        label_type (str): ["multi_class", "multi_label", "h_label", "supcon"]
+
+    Returns:
+        Dict[str, Union[int, float]]: The dictionary that includes model criteria
+    """
+    reg_config = load_regression_config(otx_dir)
+    result: Dict[str, Union[str, int, float]] = {
+        "data_path": "",
+        "model_criteria": 0,
+    }
+
+    if "anomaly" not in task_type:
+        if train_type == "" or label_type == "":
+            raise ValueError()
+        result["regression_criteria"] = reg_config["regression_criteria"][task_type][train_type][label_type]
+        result["kpi_e2e_train_time_criteria"] = reg_config["kpi_e2e_train_time_criteria"][task_type][train_type][
```

Comment:
I am not aware of the terminology. What is kpi here?

Reply:
We have several tests for maintaining the OTX

1. Pre-merge: Unit-test + Integration-test
2. Daily: E2E test
3. Weekly: Regression + KPI + Stability test

Regression: measure the model performance w.r.t. accuracy
KPI: measure the model performance w.r.t. training time and evaluation time